Sources/NIO/System.swift

```diff
@@ -363,12 +363,12 @@ internal enum Posix {
         var written: off_t = 0
         do {
             _ = try wrapSyscall { () -> ssize_t in
-                #if os(macOS)
+                #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
                     var w: off_t = off_t(count)
                     let result: CInt = Darwin.sendfile(fd, descriptor, offset, &w, nil, 0)
                     written = w
                     return ssize_t(result)
-                #else
+                #elseif os(Linux) || os(FreeBSD) || os(Android)
                     var off: off_t = offset
                     let result: ssize_t = Glibc.sendfile(descriptor, fd, &off, count)
                     if result >= 0 {
```

Comment:
FreeBSD’s sendfile is more like Darwin’s but doesn’t seem to be quite it either. Definitely not like the Linux one though. It should fall into the unsupported platform for now.

https://www.freebsd.org/cgi/man.cgi?query=sendfile&sektion=2

Reply:
@weissi I think @helje5 is right and we can do this as a follow up. WDYT ?

Reply:
Sure 👍👍

Reply:
ok let me merge then :)